appautoscaling.tf

```diff
@@ -26,3 +26,23 @@ resource "aws_appautoscaling_policy" "scale_cpu" {
     }
   }
 }
+
+resource "aws_appautoscaling_policy" "scale_mem" {
+  count              = var.launch_type == "FARGATE" ? 0 : var.autoscaling_memory ? 1 : 0
+  name               = "scale-mem"
+  policy_type        = "TargetTrackingScaling"
+  resource_id        = aws_appautoscaling_target.ecs[0].resource_id
+  scalable_dimension = aws_appautoscaling_target.ecs[0].scalable_dimension
+  service_namespace  = aws_appautoscaling_target.ecs[0].service_namespace
+
+  target_tracking_scaling_policy_configuration {
+    target_value       = var.autoscaling_target_mem
```

Comment:
If you change the variable name, needs to be changed here as well.

Reply:
Adjusted